draft-pda-core.md

```diff
@@ -291,11 +291,437 @@ Each PA task in this document is divided into three sub-protocols as follows.
 
 # PA protocols {#pa}
 
-[TODO(cjpatton)]
+This section specifies a framework for generic PA protocols. Concrete
+PA protocols are described in {{prio}} and {{hits}}.
+
+[TODO: Say that protocol messages are carried by HTTP requests and responses.]
+
+[TODO: Decide how to authenticate the leader in leader-to-helper connections.
+One option is to use client certificates for TLS; another is to have the leader
+sign its messages directly, as in Prio v2.]
+
+[TODO: Decide the "content type" for HTTP requests.]
+
+[TODO: chris-wood suggested we specify APIs for producing and consuming each of
+the messages in the protocol. Specific PA protocols would implement this API.]
+
+**Error handling.**
+In this section, we will use the verbs "abort" and "alert with `[some error
+message]`" to describe how protocol participants react to various error
+conditions. The behavior is specified in {{pa-error}}. For common errors, we may
+elide the verbs altogether and refer to to {{pa-error-common-aborts}}.
```

Comment:
This seems out of place -- was it supposed to be move to some later point in the document?

Reply:
No, it was meant to go here.

Reply:
Hmm, okay. It just feels out of place. We can always move later.